target/linux/realtek/dts-5.10/rtl838x.dtsi

```diff
@@ -45,7 +45,7 @@
 	};
 
 #define SWITCH_SFP_PORT(n, s, m) \
-	port@##n { \
+	port##n: port@##n { \
```

Comment:
If you need labels, you should be able to just prefix the macros for identical results:
`port24: SWTICH_SFP_PORT(n, s, m);`